packages/core/bootstrap/test/unit/cache-warmer.test.ts

```diff
@@ -387,6 +387,46 @@ describe('side effect tests', () => {
       })
     })
 
+    it('should create a warmup subscription  with configured interval', () => {
+      scheduler.run(({ hot, expectObservable }) => {
+        const action$ = actionStream(hot, 'a c 9s b ', {
+          a: actions.warmupSubscribed({
+            executeFn: stub(),
+            ...adapterRequest1,
+            result: adapterResult,
+          }),
+          b: actions.warmupUnsubscribed({
+            key: key1,
+          }),
+          c: actions.warmupSubscribed({
+            executeFn: stub(),
+            ...adapterRequest2,
+            result: adapterResult,
+          }),
+        })
+        const state$ = stateStream({
+          cacheWarmer: {
+            subscriptions: {
+              [key1]: { isDuplicate: false },
+              [key2]: { isDuplicate: false },
+            },
+          },
+        })
+
+        const config = { ...epicDependencies.config, warmupInterval: 8000 }
+        const output$ = warmupSubscriber(action$, state$, { config })
```

Comment:
Instead of passing through the config here could we change this to instead set the environment variable in the test?

It will not only keep the `Subscriber` cleaner, but will also mimic how users will interact with it.

Reply:
I was mainly copying how the other epics got their configs passed in, as they also got them like this. Should they remain as-is? Feels a bit weird if we're inconsistent with them.

Reply:
This looks like the proper way to inject dependencies into an epic to me, if we need users to configure the `warmupInterval`, then that configured value should be passed into the epic as a dependency when the epic is initialized